src/cfnlint/data/ExtendedSpecs/all/03_value_types.json

```diff
@@ -555,6 +649,259 @@
           "aws.config"
         ]
       },
+      "CountryCodeAlpha2": {
```

Comment:
@kddejong wondering if this should be switched to [`route53domains CountryCode` from `botocore`](https://github.com/boto/botocore/blob/2300e3a87e2835c93d832ee9c8ec7433cfa7fe18/botocore/data/route53domains/2014-05-15/service-2.json#L664)

There would be 21 Removals and 1 Addition (`"AN"`)... might need testing